docs/preview-environments/drivers.mdx

```diff
@@ -0,0 +1,112 @@
+---
+id: drivers
+slug: /preview-environments/drivers
+title: Drivers
+---
+
+import Link from "@docusaurus/Link";
+
+This section contains the list of drivers supported for writing your `porter.yaml` file.
```

Comment:
We need an overview of what a driver is - like the config and output model. For example, reading these docs it's not clear that you can reference outputs from previous resources. The discrepancy between a resource and a driver should also be clarified (not sure if that belongs here, or in the porter yaml reference doc)